### ISSUE TITLE
[Header] The menu toggle aria label string is hard coded

### ISSUE THREAD

**Issue by davidicus** [carbon-design-system/carbon-addons-iot-react#1725]
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

The `Header` component's menu toggle has an aria-label that is hard coded with "Open menu". Teams are unable to translate. We need to add this to list of translatable strings.

**Comment by davidicus** [comment]
adding a sev 2 even thought there is no workaround. We should probably align our labels to account for this.
